pkg/config/model/viper.go

```diff
@@ -211,6 +211,15 @@ func (c *safeConfig) Get(key string) interface{} {
 	return val
 }
 
+// GetAllSources returns the value of a key for each source
+func (c *safeConfig) GetAllSources(key string) ([]Source, map[Source]interface{}) {
+	values := make(map[Source]interface{})
+	for _, source := range sources {
+		values[source] = c.configSources[source].Get(key)
+	}
+	return sources, values
```

Comment:
Returning `sources`, which is a global, will allow the caller to change the internal state of the package.

Why do we need to return `sources` since they are all contained in `values` ?

Reply:
We need to return them to display them in order (as Go doesn't keep maps ordered)